Services/RotaMe.Services/RolesService.cs

```diff
@@ -24,6 +24,44 @@ public RolesService(RotaMeDbContext context, UserManager<RotaMeUser> userManager
             this.userManager = userManager;
         }
 
+        public async Task<bool> Create(RoleCreateServiceModel roleCreateServiceModel)
+        {
+            var roles = await this.context.Roles.ToListAsync();
+            if (roles.FirstOrDefault(r => r.NormalizedName == roleCreateServiceModel.NarmalizedName) == null)
+            {
+                var result = await context.Roles.AddAsync(new IdentityRole()
+                {
+                    Name = roleCreateServiceModel.Name,
+                    NormalizedName = roleCreateServiceModel.NarmalizedName
+                });
+
+                context.SaveChanges();
+
+                return false ? result == null : true;
+            } else
+            {
+                return false;
+            }
```

Comment:
Coming from PHP world if we use the statements if-else and have returned values then we can drop the else part aka.
```
if () {
  return;
}

return;
```

A further refactor could be if you revert the logic here.
```
if (! variable) {
  return false
}

some logic...

return false ? result == null : true;
```